Change date format to commonly used modern JSON one, add begin and end date params, set flag to detect if any data was generated by a file request

blackbear opened the pull request:
Hi there,
   I've been using your package for an Uploader tool that's used by the Open Humans project (formerly the Personal Genome Project at Harvard).  We needed the ability to write data out a month at a time, and to easily detect if there was any data present in a month. We also wanted a more human-parsable (and standard to modern JSON) date representation.

James Turner

onlineconvergencecorp committed:
Add start and end date

onlineconvergencecorp committed:
Add hasSamples variable

onlineconvergencecorp committed:
Fix property

onlineconvergencecorp committed:
Date format, end and start date

blackbear committed:
Fix over-ambitious test

mseemann commented:
Hi! Thx for your work!

Changing the outputformat would break the import and the lib for others. I think the outputformat should be an option (number or ISO date) and it must be ensured that the import works as well with both formats.
Right now I'm a little bit short of time to implement these features. So I would suggest that you use your fork in your project. What do you think?

Regards,
Michael Seemann

blackbear commented:
Fair enough, I figured that might be an issue. I'll implement the import side and feature setting when I have a sec.

James
On 9/26/2016 2:19:44 AM, Michael Seemann notifications@github.com wrote:
Hi! Thx for your work!
Changing the outputformat would break the import and the lib for others. I think the outputformat should be an option (number or ISO date) and it must be ensured that the import works as well with both formats.
Right now I'm a little bit short of time to implement these features. So I would suggest that you use your fork in your project. What do you think?
Regards,
Michael Seemann
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/mseemann/healthkit-sample-generator/pull/4#issuecomment-249487975], or mute the thread [https://github.com/notifications/unsubscribe-auth/AAGsEZdROG0ct6zJEMddkS64dV9y5KeTks5qt2P_gaJpZM4KGGqJ].

mseemann commented:
great!

gedankenstuecke committed:
Fix NSDateFormatter

Fix exported timestamps so that they contain 24h clock, allowing for intraday use.

blackbear committed:
Merge pull request #1 from gedankenstuecke/patch-1

Fix NSDateFormatter
